### PR TITLE
Minor en_us tweaks, update advancement translations

### PR DESCRIPTION
**Pull request by Andromander**
Just some grammar changes like capitalisation.

Also applies changes to some advancement jsons to allow the implementation of string arguments in the lang. Probably not-so-minor tweaks now.

### CODE REVIEW

File: src/main/resources/assets/twilightforest/lang/en_us.lang [hunk 1]
```diff
@@ -7,51 +7,51 @@ advancement.twilightforest.desc=Enter the mysterious magical woodlands; the Twil
...
 advancement.twilight_kill_naga=Time To Even The Scales
-advancement.twilight_kill_naga.desc=Slay the naga in its forest courtyard and obtain a Naga Scale to overcome the barrier magic surrounding the Lich's Tower.
+advancement.twilight_kill_naga.desc=Slay the Naga in its forest courtyard and obtain a Naga Scale to overcome the barrier magic surrounding the Lich's Tower.
 advancement.twilight_naga_armors=Naga Armorer
-advancement.twilight_naga_armors.desc=Craft both naga scale chest and leg armor.
+advancement.twilight_naga_armors.desc=Craft both Naga Scale chest and leg armor.
 
 advancement.twilight_kill_lich=Bring Out Your Dead
-advancement.twilight_kill_lich.desc=Slay the Lich at top of his tower and retrieve a Scepter to clear poisonous mosquitoes from the swamp, see through blinding darkness of the Dark Forest's curse, and resist the snow forest's chill.
+advancement.twilight_kill_lich.desc=Slay the Lich at top of his tower and retrieve a Scepter to clear poisonous mosquitoes from the Swamp, see through blinding darkness of the Dark Forest's curse, and resist the snow forest's chill.
```
Comment (bs2609) [inline src/main/resources/assets/twilightforest/lang/en_us.lang]: "and resist the Snowy Forest's chill"

File: src/main/resources/assets/twilightforest/lang/en_us.lang [hunk 2]
```diff
@@ -72,7 +72,7 @@ advancement.twilight_experiment_115_3.desc=eating experiment one-hundred-fifteen
 
 # Snow Forest Progression
 advancement.twilight_progress_yeti=Alpha Fur
-advancement.twilight_progress_yeti.desc=Line your garments with the soft fur from the alpha yeti, keeping you safe from the glacier's cold.
+advancement.twilight_progress_yeti.desc=Line your garments with the soft fur from the Alpha Yeti, keeping you safe from the glacier's cold.
```
Comment (bs2609) [inline src/main/resources/assets/twilightforest/lang/en_us.lang]: If this is referring to the biome, Twilight Glacier is the biome name, but if this is just referring to the glacier as an object it's fine as is.

**Review by bs2609 — COMMENTED** [review]
Looks ok, just see a couple of missed names.

File: src/main/resources/assets/twilightforest/lang/en_us.lang (hunk 2 above)
Comment (Andromander) [reply src/main/resources/assets/twilightforest/lang/en_us.lang]: That part I was a bit uncertain about, if it was just the biome being referred to as just a glacier, or the biome itself Twilight Glacier. If the latter is the case, I may also tweak the line that mentions the Twilight Swamp as just "Swamp". In any case, I'll fix that one too.

**Comment by bs2609** [comment]
It might be worth having item names, etc done using string args, like:
```
advancement.twilight_magic_map.desc=Craft the %s.
```
and then we can use the existing translations.

**Review by Tamaized — APPROVED** [review]

**Pull request merged**